lib/airbrake-ruby/code_hunk.rb

```diff
@@ -0,0 +1,51 @@
+module Airbrake
+  ##
+  # Represents a small hunk of code consisting of a base line and a couple lines
+  # around
+  class CodeHunk
+    ##
+    # @return [Integer] the maximum length of a line
+    MAX_LINE_LEN = 200
+
+    ##
+    # @return [Integer] how many lines should be read around the base line
+    INTERVAL = 3
```

Comment:
`NLINES` would be a better name. Also I suggest to lower this to `2` so we have 2 lines before and 2 lines after.

Reply:
👍